public/views/adminRights.html

```diff
@@ -0,0 +1,27 @@
+<h2>Configuration des droits des usagers</h2>
+<h4>Filtres</h4>
+<div class="row">
+    <div class="col-md-3">
+        Nom: <input type="text" class="form-control" ng-model="nameFilter" ng-change="filter('cipFilter', 'nameFilter')">
+    </div>
+    <div class="col-md-1" ng-repeat="right in rights">
```

Comment:
The upper checkboxes could go
There's already a userlist along with interactive buttons to give a summary of the granted permissions